---
title: "A Match Made in Landfills?"
subtitle: "Exploring the diversity and burden of antimicrobial resistance genes carried by white stork (Ciconia ciconia) throughout the breeding season in Madrid, Spain"
author: Raquel Francisco and Seth Latter
date: "`r Sys.Date()`"
format: 
  docx:
    toc: false
    number-sections: true
    highlight-style: github
bibliography: EW_AMR.bib
---

```{r, echo=FALSE, message=FALSE, warning=FALSE}
# load a few R packages
library(here)
library(knitr)
library(knitcitations)
library(tidyverse)
```

# Highlights

-   Our results suggest that landfills may not be as impactful as previously believed to the emergence and maintenance of AMR in this system.

-   Storks are most impacted by multi-drug resistance and ARG burden when anthropogenic waste is most heavily consumed later in the breeding season.

-   This species appears to be a good sentinel for anthropogenic impact on environment.

-   Future efforts in stork AMR research should focus on exploring the relationship between other anthropogenic environments (e.g., agricultural pastures) and health.

# Abstract

Anthropogenic environments are hotspots for the emergence and maintenance of antimicrobial resistance. Agricultural pastures and landfills are of particular interest due to their complex microbial communities and abundant wildlife visitation, which could facilitate the exchange of antimicrobial resistance genes (ARGs) via horizontal transfer. Wild birds that occupy these environments may become both reservoirs and transporters of ARGs. The White Stork is a highly urbanized wading bird that has significantly changed its ecology due to shifts in Spanish waste management toward open-air landfills. This species now heavily forages at landfills, which provide abundant food and allow for improved reproductive success. Birds that are dependent on anthropogenic resources, such as storks, provide the ideal opportunity to understand the emergence and spread of AMR. We evaluated the diversity and quantity of ARGs in storks during three periods of the breeding season (defined by distinct foraging strategies). A total of 31 nests at Prado Herrero were sampled between March-July (2020-2021). Fresh feces were collected from 31 nests to evaluate the presence of 23 important ARGs affecting eight antibiotic classes via quantitative PCR. All nests carried multiple ARGs. Over 70% of nests had multi-drug resistance to at least 3 antibiotic classes during at least one time-period. Generalized Linear Mixed Models revealed that increased diversity in antibiotic class resistance, amount of ARGs present in a sample, and multi-drug resistance were associated with increased adult age and decreased landfill use. Our results suggest that landfills may not be contributing significantly to the emergence and maintenance of AMR in this system. Little literature exists on the relationship between stork habitat selection and health outside of landfill use in Spain. Future efforts in stork AMR research should focus on exploring the relationship between agricultural land use and health.

{{< pagebreak >}}

# Introduction

## General Background Information

Antimicrobial resistance (AMR) in wildlife was recently highlighted as being a critical research need [@Dolejska2019]. While AMR is accepted as a global human health concern, it has only recently been utilized to evaluated wildlife health and anthropogenic impacts on environments (i.e., environmental health). The majority of resistance research in wildlife has been targeted towards birds, however, the importance of birds in respect to the epidemiology of AMR remains poorly understood [@Radhouani2014]. Now, a growing body of literature is trying to understand the role free-roaming birds play in the emergence, maintenance, and transmission of AMR upon the landscape.

Birds have a long history of being used as indicators for environmental and human health. For example, wading bird surveillance is commonplace in south Florida to monitor heavy metal levels in the greater everglades ecosystem. Birds are also often utilized in zoonotic pathogen surveillance (i.e., USDA sentinel chickens to monitor West Nile Virus). In recent years, a similar approach has been utilized to study AMR prevalence/impacts on different landscapes. Currently, the carriage of AMR genes presents an unknown risk to the individual birds and may have greater implications for free-living avian communities and conservation. Resistance genes are thought to alter the fitness of a pathogen [@Friedman2016], and have been correlated with increased virulence [@Escudeiro2019], in turn increasing morbidity and mortality in affected hosts. Finally, at the population management level, birds have also been held 'responsible' for the dissemination of enteric pathogens (e.g., *Escherichia coli*, *Salmonella* spp.) that have caused outbreaks in produce, yet, they are only moving these bacteria from areas already contaminated by human activities [@Hoelzer2011]. Thus, understanding how avian behaviors that overlap with risky environmental factors (e.g., foraging in agricultural areas or landfills) impact the carriage of resistance, is key because: 1) it allows us to better understand the role birds play in AMR dissemination, and 2) it informs targeted population management and conservation strategies---such as discouraging birds from utilizing certain areas.

In 2019, the USGS revealed that more than half of GPS-tracked gulls (*Larus* ssp.) had acquired antimicrobial resistant *E. coli* from landfills and disseminated it long distances to pristine, unaltered habitats [@Ahlstrom2019]. This supports the belief that birds who utilize and move between human-dominated and natural landscapes may become both reservoirs and efficient transporters of resistance determinants to natural environments.For wildlife in general, resistance prevalence tends to increase with proximity to human populations [@Niu2013]. It is believed that animals that rely on anthropogenic resources are often concentrated at high densities that promote the large-scale mixing of bacteria, encouraging horizontal gene transfer (HGT) facilitating the rise of novel resistant strains [@Wellington2013]. For instance, birds that encounter human waste (e.g., by drinking contaminated water) can acquire resistance genes through the exchange of naturally present genes in the bacteria they carry, with the genes in the waste---this can co-select for mobile genetic elements carrying multiple resistant genes [@Wellington2013]. Moreover, ecological factors such as migration and high densities during breeding season may increase acquisition rates and the dissemination of AMR . The western white stork are one such species that heavily utilizes agricultural and urbam areas, providing a unique opportunity to explore how anthropogenic land use influences the carriage of antimicrobial resistance genes.

White storks (*Ciconia ciconia*) were historically trans-Sahara migrants whose ecology has dramatically changed in response to anthropogenic pressure. Since the 1990s, Spanish white stork populations have steadily increased, due to their ability to exploit anthropogenic habitats, especially in areas with abundant resources especially open landfills [@Cramp1985; @Tortosa2002]. Currently, colonies in Madrid, Spain with increased proximity to landfills have improved breeding outcomes and nestling quality, however eggs and nestlings also have increased pollutant loads [@Vergara2006; @Jiang2013; @Saez2008; @Tortosa2002]. This has in turn affected their natural history. Subsets of the population have begun to shorten or abandon their winter migration [@Gordo2007; @Tortosa1995; @Vergara2004]. Although the conservation status of the white stork is of Least Concern in Spain, it has disappeared from large areas of its historical range in Europe and thus, the populations in Portugal and Spain are is integral to the species conservation.

Most studies on AMR surveillance in wildlife are performed by culture‐dependent methods, (e.g., the isolation of a specific pathogen such as *E. coli*) as indicators of AMR. However, since most bacteria are not culturable, the detection of AMR using traditional methods might not be representative of the whole resistant microbiota, sometimes referred to as the resistome [@Surette2017]. Real time PCR (rtPCR), a technique not dependent on culture, offers that ability to quantify the presence and abundance of antimicrobial resistance genes (ARGs), in white storks. This technique has recently been used in three studies to investigate the presence and amount of ARGs in the environment (e.g., soil and manure) and wildlife (e.g., Galapagos tortoise, gulls, guignas) [@Esperon2014; @NietoClaudin2019; @Sacristan2020]. In this study we will explore the ARG load and diversity in a population of white stork in Madrid Spain during the 2020 and 2021.

## Objective

Evaluate both antimicrobial gene resistance diversity and burden in white storks that utilize landfills and natural areas over the breeding season of years 2020 to 2021.

## Hypotheses

Increased landfill use will increase both the diversity and burden of antimicrobial resistance genes in white stork.

Nest success will be negatively affected by antimicrobial gene burden and multi-drug resistance (i.e., resistance to 3 or more drug classes).

{{< pagebreak >}}

# Methods

## Data aquisition

### Description of data and data source

We have 126 observations taken over a period of 2 years. We are evaluating for the presence and burden of antimicrobial resistance genes in white stork feces. A codebook containing descriptions of variables and their abbreviations can be found in the raw data folder, located within the project repository.

### Experimental Design

*Study Area* --- This study took place in Prado Herrero, a private cattle ranch located northwest of metropolitan Madrid and is surrounded by agriculture (e.g. beef cattle, cereal grains, legumes, and forage plants. Prado Herrero is located within a nationally protected area (Cuenca Alta del Manzanares Regional Park) and is just north of Santillana Reservoir. This reservoir that was declared an Important Bird Area by Regional Catalogue of Reservoirs and Wetlands of the Community of Madrid due to the numerous resident and migratory species that utilize this water source. This cattle ranch has supported a productive white stork rookery where storks have been banded and monitored by biologists at UCM for over 20 years [@Aguirre2009]. During the 2020 to 2021 breeding season, between the months of March to June, stork nests were identified, marked, and monitored for productivity. Of marked nests between the 2020 and 2021 breeding seasons, 31 with banded adults were used both years to lay eggs successfully. All 31 nests were located in ash trees (*Fraxinus angustifolius*) found within the cattle pasture. Storks that breed within the Prado Herrero rookery are known to utilize Colmenar Viejo Landfill which is located approximately 12km southeast. Colmenar Viejo is an open-air landfill and it is second largest of it's kind in the Madrid region [@Lopez-Garcia2021].

*Sample Collection* --- Between March to May 2020 and 2021, we collected feces from marked nests with banded adults in known breeding pairs at 3 points of the breeding season; (1) an adult sample during incubation, (2) an early juvenile sample during the first two weeks of the chicks life when adults are believed to forage on natural sources, and (3) a late juvenile sample after chicks were past two weeks of age when adults forage on anthropogenic resources. Nests were visited in the late mornings and approximately one gram of fresh feces was collected from the perimeter of the nest structure into a sterile Eppendorf tube. Samples were maintained cold in a portable cooler with frozen gel packs and frozen in a −20°C freezer within 4 hours of collection and processed at a later date.

*Ethics statement*: All animal handling was authorized by Cumunidad de Madrid: Consejeria de Medio Ambiente, Administracion Local y Ordenacion de Territorio. The permit number is D.N.I. nº 07.239.972-D.

### Molecular analysis of ARGs

We performed total DNA extraction directly from fecal samples, by using a pressure filtration technique (QuickGene DNA Tissue Kit S, Fujifilm, Japan) following the manufacturer's instructions. The 16S rRNA gene was amplified in each DNA sample by real time PCR (rtPCR) in 10-fold dilutions of extracted samples, according to @Jiang2013. A DNA sample was considered validated when a ten-fold dilution showed a cycle threshold (Ct) less than 25 [@Esperon2020]. Once validated, we analyzed DNA samples by with a panel of 21 different ARGs encoding resistance to eight different antimicrobial classes: tetracyclines (tet(A), tet(B), tet(Y), tet(K), tet(M), tet(Q), tet(S), and tet(W)), sulfonamides (sulI and sulII), aminoglycosides (str and aadA), phenicols (catI and catII), macrolides (ermB and ermF), quinolones (qnrS and qnrB), betalactams (blaTEM and mecA), and polymyxins (mcr-1). All rtPCR reactions utilized premade gelled format 96-well plates (Biotools, B &M Labs, S.A., Madrid, Spain), with the exception of blaTEM and mecA genes which used the Sybr GreenTM and TaqManTM probe, respectively. Our thermal cycle was the same for all the rtPCR reactions \[6′ 95 °C, 40× (10″ 95 °C, 30″ 60 °C)\], with alignment and extension in the same step, at constant temperature of 60 °C. A melting curve step was performed at the end of the qPCR reaction to validate the authenticity of the positive [@NietoClaudin2019]. We quantified the relative burden of each gene for each sample via the cycle threshold (Ct) for the 16S rRNA and the Ct value of the individual ARG using a previously published formula in @Esperon2020.

## Data import and cleaning

Raw data was collected during both field seasons (2020 and 2021) and maintained on an excel file. This excel file was amended in 2021 to include the laboratory results from the AMR qPCR. The raw excel file can be found in the "1 Data Cleaning Script" folder in this [projects repository](https://github.com/rfranci3/SETHLATTERRAQUELFRANCISCO-MADA-project). Cleaned data was evaluated for normality and each variable was standardized and checked correlative relationships. All variables were kept as they did not appear strongly correlated.

### Statistical analysis

Presence absence ARG results obtained from the fecal samples between 2020 to 2021 were used for simple summary statistics. Samples were classified as "multiresistant" if they were resistant to three or more of the 8 antibiotic classes that we evaluated for in this study [@Blanco-Pena2017]. In addition, we applied the following formula to estimate the percentage of bacteria harboring ARGs: x = 10\[2+0.33(ct16S-ctARG)\], where x individual percent gene burden in the sample (i.e., the estimated number of copies of the gene present per reaction). Results were expressed in log10, ranging from −8 ( zero to a negligible amount of the bacteria in the sample carried an ARG) to 2 (all or 100% of the bacteria in the sample carried an ARG). The inverse Log10 was then applied to results so they could be totaled and used to evaluate total gene burdens across sampling periods.

Several linear mixed models (LMM) were constructed to evaluate multi-resistance and ARG burden as response variables with nest as a random factor. Covariates considered with each response variable included adult age, adult mean land fill use index (LUI), sample period (as described above), and nest success. Landfill use was quantified by physically observing a banded stork at Colmenar Viejo during weekly visits from March to June in 2021. The LUI was calculated as the number of observations of one particular bird within the total number of visits to landfill per year [@Lopez-Garcia2021]. If a banded adult was not seen at the landfill during the breeding season, they were assigned a LUI of 0, suggestive of no resource provisioning at the landfill. All covariates were evaluated for correlation, no covariates were correlated with all the Spearman's correlation coefficients (rho) \< 0.5 and the p \> 0.05. All continuous variables (LUI, age, and nest success) were then standardized prior to analysis.

All models were constructed with only 2021 data, as the COVID-19 pandemic prohibited the collection of LUI data in 2020. Models were built and fitted to data using the statistical package tidymodels in Program R (R version 4.2.1, www.r-project.org).

{{< pagebreak >}}

# Results

## Exploratory preliminary analysis

A total of 126 fecal samples were collected from white storks from 2020 to 2021. Of note, resistance genes to tetracyclines and betalactamines were identified in 50 (39.68%) and 73 (57.94%) fecal samples, respectively. Multi-drug resistance was identified in 39 (30.95%) samples. Mean landfill use index was 0.14 across all samples from 2021, and mean nest success was 55.95% across both sampling periods.

Cleaned data was then visually explored to evaluate trends. Notably, it did not seem to appear that multi-drug resistance played a factor in nest success (Figure 2). However, sampling periods did appear important to the amount of ARG burden found in white stork feces (Figure 3). The most notable trend in the data however, was the appearance that multi-drug resistance in fact declined with the increase of landfill use by the adult white storks (Figure 4).

![Figure 2: Multi-drug resistance and nest success (i.e., number of chicks fledged) for each nest during the 2020 and 2021 white stork breeding seasons.](Images/Prelim_LUIvsNS.tiff)

![Figure 3: Antimicrobial Drug Resistence burden across all three sampling periods during the 2020 and 2021 white stork breeding seasons.](Images/Prelim_BurdenvsSamp.tiff)

![Figure 4: Multi-drug resistance and Landfill Use Index for each nest during the 2020 and 2021 white stork breeding seasons.](Images/Prelim_LUIvsMD.tiff)

## Full Statistical Analysis

![Figure 5: Multi-drug resistance as predicted by the variables landfill use index and age during the 2020 and 2021 white stork breeding seasons. The gray is a 95% CI.](Images/LUIvsMDR_LME.tiff)

Binomial generalized linear mixed models (GLMMs) were used to predict multi-drug resistance (MDR) in white storks, with nest identification classified as a random effects parameter. Of these, the highest performing model contained the predictors landfill use index (LUI) and age (Table 1). In this model, an increase in the age of the bird was associated with a higher likelihood of MDR being present (Table 2), while an increase in LUI was found to be associated with a decreased likelihood of MDR presence. A second competitive model, with a 97.20% performance score, contained only age as the predictor. In this analysis, several models where overfit, not including the top two models. However, this is likely due to the small sample size and the restricted range of the standardized variables.

```{r}
#| label: Table2
#| tbl-cap: "Parameter estimates for fixed and random effects parameters and standard error of the highest performing model predicting multi-drug resistance in White Storks in Madrid, Spain."
#| echo: FALSE
GLMERmodel1 = readRDS(here("4 Results", "Models", "GLMERmodel1.rds"))
knitr::kable(GLMERmodel1)
```

![Figure 6: Visual model performance analysis of the all the GLMMs used to evaluate Multi-Drug Resistance (i.e., resistance to 3 or more drug classes). Covariates used includes: Age, Sampling Period, Landfill Use, and Nest Success. Individual Nests were the fixed variable. Due to small sample sizes AICc was viewed as the most appropriate metric, however AIC, BIC, RSME, R2, and Sigma are displayed.](Images/glmer_indices.tiff)

```{r}
#| label: tbl-resulttable2
#| tbl-cap: "Model AICc Tables."
#| echo: FALSE
ModelAIC1 = readRDS(here("4 Results", "Tables", "ModelAIC1.rds"))
knitr::kable(ModelAIC1)

```

![Figure 7: Anti-microbial resistance gene burden of each nest as predicted by the variables landfill use index and sampling period during the 2020 and 2021 white stork breeding seasons. The gray is a 95% CI.](Images/LUIvsBurden_LME.tiff)

Linear mixed models (LMMs) were used to predict total antimicrobial gene burden in white storks, with nest identification again classified as a random effects parameter. The global model, containing the variables landfill use index, age, sample period, and nest success, was the highest performing model (Table 3). In this model, each predictor variable was positively correlated with antimicrobial gene burden (Table 4). The next highest performing model contained age as a single predictor, with a model performance score of 86.39%. As with the GLMM analysis, several models, including the top model, where overfit, likely due to similar reasons.

```{r}
#| label: Table4
#| tbl-cap: "Parameter estimates for fixed and random effects parameters and standard error of the highest performing model predicting total antimicrobial gene burden in White Storks in Madrid, Spain."
#| echo: FALSE
GLMERmodel1 = readRDS(here("4 Results", "Models", "LMERmodel2.rds"))
knitr::kable(GLMERmodel1)
```

![Figure 8: Visual model performance analysis of the all the LMMs used to evaluate Antimicrobial Resistance Gene Burden. Covariates used includes: Age, Sampling Period, Landfill Use, and Nest Success. Individual Nests were the fixed variable. Due to small sample sizes AICc was viewed as the most appropriate metric, however AIC, BIC, RSME, R2, and Sigma are displayed.](Images/lmer_indices.tiff)

```{r}
#| label: Table3
#| tbl-cap: "Small sample adjusted Akaike's Information Criteria (AICc) model weights and performance scores for linear mixed models predicting total antimicrobial gene burden in White Storks (Ciconia ciconia) in Madrid, Spain. "
#| echo: FALSE
ModelAIC2 = readRDS(here("4 Results", "Tables", "ModelAIC2.rds"))
knitr::kable(ModelAIC2)

```

{{< pagebreak >}}

# Discussion

## Summary and Interpretation

-   As multi-drug resistance increases throughout the breeding season, mean LUI decreases and nest success decreases. There is a positive effect between increased LUI and nest success [@Lopez-Garcia2021]. Intuitively, this makes some sense due to MDR being associated with increased anthropogenic exposure, heavy metal exposure, and increased virulence of pathogenic bacteria. More targeted research would be needed to better understand how MDR may effect nest success long-term.

-   Resistance gene burden appears to increase as mean LUI decreases and age increases. Most of the total burden is attributed to blaTEM, a common resistance gene associated with beta-lactam resistance. Beta-lactam resistance is one of the most studied AMRs in wildlife. Drugs within the Beta-lactam class are often first-line antibiotics, such as penicillin, and are active against many gram-positive, gram-negative, and anaerobic organisms. At this point, beta-lactam resistance is thought to be prevalent on most landscapes due to human or livestock encroachment and thus is often used as a measure of anthropogenic impact. While sampling periods did not explain burden, the top blaTEM model did show a trend in burden increasing from the 2nd sampling period to the 3rd sampling period.This may be related to sheer increase of foraging required to feed \~3 to 4 kilogram fledglings.This foraging could have occurred in areas other then the landfill, such as neighboring cattle pastures which may also be sources for ARGs.

## Strengths and Limitations

-   While none of our models resulted in statistically significant data, valuable trends can be seen in respect to the relationship between breeding season events, AMR trends, and landfill use. Hopefully, these trends can foster future research to consider expanding beyond the hypothesis that landfills are the risky place for wildlife to acquire ARGs.

-   The dataset utilized for the models was much smaller then previously envisioned due to COVID-19 barring landfill data collection throughout the entirety of 2020. Perhaps with more observations, the results of the models may have been stronger and variables may have been more normally distributed. While model residuals should be normally-distributed, model performance metrics indicated polarization in the residuals in several of our top model, which can lead to under- and over-fitting data. Moving forward, it may be best to transform our standardized variables using a log transformation to attempt to achieve a more normal distribution for analysis, especially in regard to our GLMMs.  

## Conclusions

LUI appears to be correlated with higher levels of AMR gene burden in storks. As LUI increases thorough the breeding season [@Bialas2020] resistance gene burden also increases with beta-lactam resistance contributing to the majority of the burden. However, multi-drug resistance appears to decrease as LUI increases, thus it is likely that storks are being exposed to antimicrobial resistance genes at other foraging areas (urban centers, agricultural pastures, etc.). Our results suggest that landfills may not be contributing significantly to the emergence and maintenance of AMR in this system. Little literature exists on the relationship between stork habitat selection and health outside of landfill use in Spain. Future efforts in stork AMR research should focus on exploring the relationship between agricultural land use and health.

{{< pagebreak >}}

# References
